barretenberg/cpp/src/barretenberg/vm2/simulation/range_check.test.cpp

```diff
@@ -0,0 +1,29 @@
+#include "barretenberg/vm2/simulation/range_check.hpp"
+
+#include <gtest/gtest.h>
+
+#include "barretenberg/vm2/simulation/events/event_emitter.hpp"
+
+namespace bb::avm2::simulation {
+namespace {
+
+TEST(AvmSimulationRangeCheckTest, AssertRange)
```

Comment:
Could you add a test where you
* create a single row with `clk = 1`. this is to force a row to be in the trace. I actually want a completely empty row but it doesnt matter.
* check the whole relation and expect no failures
?

This is to test that all relations pass in rows fulls of 0. We probably need to add this test to every gadget.